### DISCUSSION TITLE
Anypoint Code Builder not sycn with Design Center

### DISCUSSION THREAD

**Discussion by kalyaninamasani55** [coder/code-server#6758] [Q&A]
I have created an API specification in Anypoint Code Builder but was not able to see them in design center, I was able to sync with git, can also able to publish to exchange though.
I checked business groups, permissions, all seems to be right. 
Is there anything I'm missing out here? Thanks in advance.

**Comment by code-asher** [comment]
It looks like you might have the wrong repository, this is for code-server (VS Code in the browser).
